it/uk/gov/hmrc/traderservices/controllers/TraderServicesRouteOneISpec.scala

```diff
@@ -245,7 +248,7 @@ class TraderServicesRouteOneISpec
         result.status shouldBe 201
         result.json.as[JsObject] should (
           haveProperty[String]("correlationId", be(correlationId)) and
-            haveProperty[String]("result", be("PCE201103470D2CC8K0NH3"))
+            haveProperty[TraderServicesResult]("result")
```

Comment:
here and below we should still check the returned value of created case reference number; you can nest `haveProperty` assertions

Reply:
`haveProperty[JsObject]("result", haveProperty[String]("caseId", be("PCE201103470D2CC8K0NH3")))`

Reply:
PR updated